crates/fsrt/src/main.rs

```diff
@@ -653,10 +687,15 @@ pub(crate) fn scan_directory<'a>(
         used_graphql_perms.extend_from_slice(&graphql_perms_defid);
         used_graphql_perms.extend_from_slice(&graphql_perms_varid);
 
+        let oauth_scopes: Vec<&&String> = used_graphql_perms
+            .iter()
+            .map(|val| scope_name_to_oauth.get(&val.as_str()).unwrap())
```

Comment:
Would there ever be a case where there's a possible GraphQL perm detected that's not in the scope_name_to_oauth map? If so, we might want to just skip over it, but log a warning instead of crashing.

Reply:
Also, this might be better as a hashset of `&str`, since the Vec might get big if there are a lot of individual api usages.

Reply:
Sounds good. Made this a hashset. I think using `String` is a limitation of the API.